bin/node-pg-migrate

```diff
@@ -165,6 +151,11 @@ const { argv } = yargs
     type: 'string',
   })
 
+  .option(envPathArg, {
+    describe: 'path to the .env file that should be used for configuration',
```

Comment:
**nitpick (non-blocking):** Most of the `describe` seems to start with an upper case letter  
Could you change these to always start with an upper case letter? :slightly_smiling_face:  
(So e.g. also the one from `tsconfigArg`)

Reply:
Sure, did not notice this, will align to the style.